src/guide/essentials/template-refs.md

```diff
@@ -263,6 +264,6 @@ export default {
 }
 ```
 
-In the above example, a parent referencing this component via template ref will only be able to access `publicData` and `publicMethod`.
-
+Yuqoridagi misolda, a parent referencing this component via template ref will only be able to access `publicData` and `publicMethod`.
```

Comment:
Tarjima to'liq emas
![image](https://user-images.githubusercontent.com/12369295/172148141-5e700de7-4c3a-4616-86d9-2d1a17061bd2.png)

Reply:
To'g'rilandi.